builder/build-workstation-template

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Wrapper script to manage the TemplateVM build process.
+# Clones the qubes-builder repo, as well as the securedrop-workstation repo,
+# and generates an RPM that can be installed in dom0 to support new TemplateVMs.
+set -e
+set -u
+set -o pipefail
+
+repo_root="$(git rev-parse --show-toplevel)"
+build_dir="${repo_root}/builder"
+qubes_builder_dir="${build_dir}/qubes-builder"
+sd_template_dir="${build_dir}/qubes-template-securedrop-workstation"
+
+# Heavy-handed, but ensures build is clean, and also sidesteps idempotence
+# issues with the git clone/checkout commands.
+rm -rf "${qubes_builder_dir}" "${sd_template_dir}"
```

Comment:
The `rm -rf` here doesn't provide sufficient cleanup. If a build fails, and the chroot is provisioned, then the `rm -rf` will nuke all files _other_ than the chroot (which is root-owned). We should run `make -C "${qubes_builder_dir}" clean-chroot` or similar in order to tear down the chroot responsibly, _then_ proceed with removing the dir.